--- graphviz-2.50.0/cmd/lefty/tbl.c.orig	2021-12-04 21:10:28.000000000 +0100
+++ graphviz-2.50.0/cmd/lefty/tbl.c	2024-07-27 15:26:26.654224436 +0200
@@ -170,7 +170,7 @@
             cp2[i] = cp[i];
             if (cp2[i].next != C_NULL)
                 cp2[i].next -= ci;
-            s = &cp[i].u.s;
+            s = cp[i].u.s;
             while (*s)
                 s++;
             cn = (long) (s - (char *) &cp[i]) / sizeof (Code_t);
