--- graphviz-2.30.1/configure.ac.orig	2013-03-30 13:33:33.991726307 +0100
+++ graphviz-2.30.1/configure.ac	2013-03-30 13:35:38.171723708 +0100
@@ -1567,7 +1567,7 @@
 	    if test "x$DARWIN" == "xyes"; then
 	      RUBY_CFLAGS=`echo $RUBY_CFLAGS | sed 's/powerpc/universal/'`
 	    fi
-            RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` `$RUBY -rrbconfig -e \"puts Config::CONFIG[['LIBRUBYARG_SHARED']]\"`"
+            RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` $($RUBY -rrbconfig -e "puts Config::CONFIG[['LIBRUBYARG_SHARED']]")"
             RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb vendorarchdir`"
 	    if test "x$RUBY_INSTALL_DIR" == "x"; then
 	  	  RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb sitearchdir`"
--- graphviz-2.30.0/config/config_ruby.rb.orig	2013-01-19 19:53:28.134176235 +0100
+++ graphviz-2.30.0/config/config_ruby.rb	2013-01-19 19:58:10.874170341 +0100
@@ -8,6 +8,10 @@
     puts Config::expand(CONFIG["archdir"])
 when "lib"
     puts Config::expand(CONFIG["libdir"])
+when "includes"
+    puts Config::expand(CONFIG["rubyhdrdir"])
+when "archincludes"
+    puts Config::expand(CONFIG["rubyhdrdir"]+"/"+CONFIG["arch"])
 when "vendorarchdir"
     puts Config::expand(CONFIG["vendorarchdir"])
 when "sitearchdir"
