diff -urN graphviz-2.14.1/configure.ac graphviz-2.14.1.new/configure.ac
--- graphviz-2.14.1/configure.ac	2008-05-25 17:49:41.650208531 +0200
+++ graphviz-2.14.1.new/configure.ac	2008-05-25 17:51:30.172271450 +0200
@@ -11,18 +11,18 @@
 # libtool shared library version
 
 # Increment if the interface has additions, changes, removals.
-LT_CURRENT=4
+LTVER_CURRENT=4
 
 # Increment any time the source changes; set to
 # 0 if you increment CURRENT
-LT_REVISION=0
+LTVER_REVISION=0
 
 # Increment if any interfaces have been added; set to 0
 # if any interfaces have been removed. removal has
 # precedence over adding, so set to 0 if both happened.
-LT_AGE=0
+LTVER_AGE=0
 
-VERSION_INFO="$LT_CURRENT:$LT_REVISION:$LT_AGE"
+VERSION_INFO="$LTVER_CURRENT:$LTVER_REVISION:$LTVER_AGE"
 AC_SUBST(VERSION_INFO)
 
 dnl ===========================================================================
@@ -38,10 +38,10 @@
 # This nonsense drives me crazy!!
 case "${host_os}" in
   *irix* | *nonstopux* )
-    GVPLUGIN_VERSION=`expr $LT_CURRENT + 1`
+    GVPLUGIN_VERSION=`expr $LTVER_CURRENT + 1`
     ;;
   * )
-    GVPLUGIN_VERSION=$LT_CURRENT
+    GVPLUGIN_VERSION=$LTVER_CURRENT
     ;;
 esac
 AC_DEFINE_UNQUOTED(GVPLUGIN_VERSION,$GVPLUGIN_VERSION,[Compatibility version number for plugins.])
