diff -urN aa/graphviz-1.11/dotneato/Makefile.am graphviz-1.11/dotneato/Makefile.am
--- aa/graphviz-1.11/dotneato/Makefile.am	2003-12-29 17:36:57.000000000 +0100
+++ graphviz-1.11/dotneato/Makefile.am	2004-02-29 20:07:50.940349608 +0100
@@ -48,7 +48,7 @@
 	$(top_builddir)/pathplan/libpathplanconvenience.la \
 	$(top_builddir)/graph/libgraphconvenience.la \
 	$(top_builddir)/cdt/libcdtconvenience.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 dot_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -58,7 +58,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 neato_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -69,7 +69,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 twopi_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -81,7 +81,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 fdp_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -93,7 +93,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 circo_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -105,7 +105,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 dotmemtest_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -115,7 +115,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 neatomemtest_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -126,7 +126,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
 	$(top_builddir)/graph/libgraph.la \
 	$(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 twopimemtest_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -138,7 +138,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
         $(top_builddir)/graph/libgraph.la \
         $(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 fdpmemtest_LDADD = \
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
@@ -150,7 +150,7 @@
 	$(top_builddir)/pathplan/libpathplan.la \
         $(top_builddir)/graph/libgraph.la \
         $(top_builddir)/cdt/libcdt.la \
-	$(top_builddir)/gd/libgdconvenience.la
+	-lgd
 
 EXTRA_DIST = $(man_MANS) XFIG_FORMAT3.2.txt nmakefile Makefile.old \
 	dotneato-config.in awk
diff -urN aa/graphviz-1.11/gdtclft/Makefile.am graphviz-1.11/gdtclft/Makefile.am
--- aa/graphviz-1.11/gdtclft/Makefile.am	2003-11-08 12:38:08.000000000 +0100
+++ graphviz-1.11/gdtclft/Makefile.am	2004-02-29 20:08:20.131911816 +0100
@@ -21,8 +21,8 @@
 
 libgdtclft_la_LIBADD = \
 	$(top_builddir)/tclhandle/libtclhandle.la \
-	$(top_builddir)/gd/libgdconvenience.la \
-	$(top_builddir)/tclstubs/libtclstubs.la
+	$(top_builddir)/tclstubs/libtclstubs.la \
+	-lgd
 
 all-local: pkgIndex.tcl
 
diff -urN aa/graphviz-1.11/Makefile.am graphviz-1.11/Makefile.am
--- aa/graphviz-1.11/Makefile.am	2003-11-26 14:01:45.000000000 +0100
+++ graphviz-1.11/Makefile.am	2004-02-29 20:09:00.037845192 +0100
@@ -5,7 +5,7 @@
 pkgindexdir = $(libdir)/@PACKAGE@
 pkgindex_DATA = pkgIndex.tcl
 
-SUBDIRS = cdt agraph agutil doc pathplan graph gd dotneato \
+SUBDIRS = cdt agraph agutil doc pathplan graph dotneato \
 	dotty graphs lefty lneato ns \
 	tclstubs tclhandle gdtclft tcldot tclpathplan \
         tkstubs tkspline tools contrib dynagraph windows
diff -urN aa/graphviz-1.11/tcldot/Makefile.am graphviz-1.11/tcldot/Makefile.am
--- aa/graphviz-1.11/tcldot/Makefile.am	2003-12-01 16:15:16.000000000 +0100
+++ graphviz-1.11/tcldot/Makefile.am	2004-02-29 20:08:29.722453832 +0100
@@ -44,9 +44,9 @@
         $(top_builddir)/pathplan/libpathplanconvenience.la \
         $(top_builddir)/graph/libgraphconvenience.la \
 	$(top_builddir)/cdt/libcdtconvenience.la \
-	$(top_builddir)/gd/libgdconvenience.la \
 	$(top_builddir)/gdtclft/gdtclft.lo \
-	$(top_builddir)/tclstubs/libtclstubs.la
+	$(top_builddir)/tclstubs/libtclstubs.la \
+	-lgd
 
 all-local: pkgIndex.tcl
 
diff -urN aa/graphviz-1.11/tools/src/Makefile.am graphviz-1.11/tools/src/Makefile.am
--- aa/graphviz-1.11/tools/src/Makefile.am	2004-02-23 21:28:52.000000000 +0100
+++ graphviz-1.11/tools/src/Makefile.am	2004-02-29 20:08:42.258548056 +0100
@@ -114,9 +114,9 @@
 	$(top_builddir)/dotneato/common/libcommonconvenience.la \
 	$(top_builddir)/dotneato/gvrender/libgvrenderconvenience.la \
 	$(top_builddir)/pathplan/libpathplanconvenience.la \
-	$(top_builddir)/gd/libgdconvenience.la \
 	$(top_builddir)/graph/libgraphconvenience.la \
-	$(top_builddir)/cdt/libcdtconvenience.la
+	$(top_builddir)/cdt/libcdtconvenience.la \
+	-lgd
 
 dijkstra_SOURCES = dijkstra.c
 
