--- graphviz-2.26.3/configure.ac.orig	2010-10-22 09:01:47.799741043 +0200
+++ graphviz-2.26.3/configure.ac	2010-10-22 23:17:41.319743558 +0200
@@ -1404,10 +1404,10 @@
       if test "x$RUBY" = "x"; then
 	use_ruby="No (ruby not available)"
       else
-        RUBY_INCLUDES="-I`$RUBY $srcdir/config/config_ruby.rb INCLUDES`"
+        RUBY_INCLUDES="-I`$RUBY $srcdir/config/config_ruby.rb INCLUDES` -I`$RUBY $srcdir/config/config_ruby.rb archincludes`"
 	# hack for powerpc-darwin8 (10.4)
 	RUBY_INCLUDES=`echo $RUBY_INCLUDES | sed 's/powerpc/universal/'`
-        RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` `$RUBY -rrbconfig -e \"puts Config::CONFIG[['LIBRUBYARG_SHARED']]\"`"
+        RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` $($RUBY -rrbconfig -e "puts Config::CONFIG[['LIBRUBYARG_SHARED']]")"
         RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb archsitelib`"
         save_CPPFLAGS=$CPPFLAGS
         CPPFLAGS="$CPPFLAGS $RUBY_INCLUDES"
--- graphviz-2.26.3/config/config_ruby.rb.orig	2005-03-26 19:49:00.000000000 +0100
+++ graphviz-2.26.3/config/config_ruby.rb	2010-10-22 23:22:53.627734339 +0200
@@ -5,7 +5,9 @@
 
 case ARGV[0]
 when "INCLUDES"
-    puts Config::expand(CONFIG["archdir"])
+    puts Config::expand(CONFIG["rubyhdrdir"])
+when "archincludes"
+    puts Config::expand(CONFIG["rubyhdrdir"]+"/"+CONFIG["arch"])
 when "lib"
     puts Config::expand(CONFIG["libdir"])
 when "archsitelib"
