--- graphviz-2.40.1-root-comp/usr/include/graphviz/geom.h.orig	2017-06-01 20:57:43.249931759 +0200
+++ graphviz-2.40.1-root-comp/usr/include/graphviz/geom.h	2017-06-01 20:57:49.146598357 +0200
@@ -17,7 +17,9 @@
 #ifndef GV_GEOM_H
 #define GV_GEOM_H
 
+#ifdef GRAPHVIZ_GEOM_WITH_MACROS
 #include "arith.h"
+#endif
 
 #ifdef __cplusplus
 extern "C" {
@@ -34,6 +36,7 @@
 
 typedef struct { pointf LL, UR; } boxf;
 
+#ifdef GRAPHVIZ_GEOM_WITH_MACROS
 
 /* true if point p is inside box b */
 #define INSIDE(p,b)	(BETWEEN((b).LL.x,(p).x,(b).UR.x) && BETWEEN((b).LL.y,(p).y,(b).UR.y))
