--- graphviz-2.38.0/plugin/ming/Makefile.am.orig	2014-04-13 22:40:25.000000000 +0200
+++ graphviz-2.38.0/plugin/ming/Makefile.am	2014-04-21 09:11:51.695995652 +0200
@@ -28,7 +28,7 @@
 
 libgvplugin_ming_la_LDFLAGS = -version-info @GVPLUGIN_VERSION_INFO@
 libgvplugin_ming_la_SOURCES = $(libgvplugin_ming_C_la_SOURCES)
-libgvplugin_ming_la_LIBADD = @MING_LIBS@ @MATH_LIBS@
+libgvplugin_ming_la_LIBADD = $(top_builddir)/lib/gvc/libgvc.la @MING_LIBS@ @MATH_LIBS@
 
 if WITH_WIN32
 libgvplugin_ming_la_LDFLAGS += -no-undefined
--- graphviz-2.47.2/plugin/ming/gvrender_ming.c.orig	2021-05-27 02:56:39.000000000 +0200
+++ graphviz-2.47.2/plugin/ming/gvrender_ming.c	2022-03-24 16:55:38.941518316 +0100
@@ -13,6 +13,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <gvc/gvplugin_render.h>
+#include "gvcint.h"
+#include "gvcproc.h"
 
 /* from config.h - would conflict with ming.h definitions */
 #undef PACKAGE
@@ -83,7 +85,6 @@ static void ming_end_page(GVJ_t * job)
     SWFMovie_nextFrame(movie);
 }
 
-extern char* gvconfig_libdir(void);
 #define FONT "Bitstream_Vera_Serif.fdb"
 
 static void ming_textspan(GVJ_t * job, pointf p, textspan_t * span)
@@ -100,7 +101,7 @@ static void ming_textspan(GVJ_t * job, p
 
 /* FIXME - hardcoded to a Times-like font */
     if (font == NULL) {
-    	libdir=gvconfig_libdir();
+    	libdir=gvconfig_libdir(job->gvc);
 	font_file_name = malloc(strlen(libdir)+strlen(FONT)+2);
 	strcpy(font_file_name, libdir);
 	strcat(font_file_name, "/");
