--- graphviz-2.50.0/tclpkg/gv/gv_php_init.c.orig	2022-09-11 19:23:28.076467439 +0200
+++ graphviz-2.50.0/tclpkg/gv/gv_php_init.c	2022-09-11 19:32:35.430168837 +0200
@@ -24,12 +24,14 @@
 static size_t gv_string_writer (GVJ_t *job, const char *s, size_t len)
 {
     (void)job;
+    TSRMLS_FETCH();
     return PHPWRITE(s, len);
 }
 
 static size_t gv_channel_writer (GVJ_t *job, const char *s, size_t len)
 {
     (void)job;
+    TSRMLS_FETCH();
     return PHPWRITE(s, len);
 }
 
