--- graphviz-2.28.0/configure.ac.orig	2011-05-17 20:36:04.898079515 +0200
+++ graphviz-2.28.0/configure.ac	2011-05-17 21:16:08.718160002 +0200
@@ -676,7 +676,7 @@
       if test "x$GUILE" = "x"; then
         use_guile="No (guile not available)"
       else
-        GUILE_VERSION=`$GUILE --version | head -1 | cut -d ' ' -f 2`
+        GUILE_VERSION=$($GUILE --version | head -1 | sed -e 's/.* \([[0-9.]*]\)$/\1/')
         GUILE_VERSION_MAJOR=`echo $GUILE_VERSION | cut -d '.' -f 1`
         GUILE_VERSION_MINOR=`echo $GUILE_VERSION | cut -d '.' -f 2`
         if test $GUILE_VERSION_MAJOR -lt 2; then
@@ -691,12 +691,15 @@
         if test "x$GUILE" = "x"; then
 	  use_guile="No (guile is too old)"
         else
-          GUILE_INCLUDES=
-          GUILE_LIBS=
+          GUILE_INCLUDES=$(guile-config compile)
+          GUILE_LIBS=$(guile-config link)
+	   save_CPPFLAGS="$CPPFLAGS"
+	   CPPFLAGS="$CPPFLAGS $GUILE_INCLUDES"
           AC_CHECK_HEADER(libguile.h,,[
             use_guile="No (guile header missing)"
             GUILE=
           ])
+          CPPFLAGS="$save_CPPFLAGS"
           if test "x$GUILE" != "x"; then
 	    use_guile="Yes"
             AC_SUBST(GUILE_INCLUDES)
