--- graphviz-2.20.3/configure.ac.orig	2008-10-14 17:15:51.463774247 +0200
+++ graphviz-2.20.3/configure.ac	2008-10-14 17:16:28.461231498 +0200
@@ -721,7 +721,7 @@
     if test `$SWIG -help 2>&1 | $GREP -c '\-lua *- Generate'` = 0; then
       use_lua="No (swig does not support -lua option)"
     else
-      AC_CHECK_PROG(LUA,lua,lua)
+      AC_CHECK_PROG(LUA,lua51,lua)
       if test "x$LUA" = "x"; then
         use_lua="No (lua not available)"
       else
