--- graphviz-2.30.0/plugin/ming/Makefile.am.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/plugin/ming/Makefile.am	2013-01-20 17:23:58.735895674 +0100
@@ -28,7 +28,7 @@
 
 libgvplugin_ming_la_LDFLAGS = -version-info @GVPLUGIN_VERSION_INFO@
 libgvplugin_ming_la_SOURCES = $(libgvplugin_ming_C_la_SOURCES)
-libgvplugin_ming_la_LIBADD = @MING_LIBS@ @MATH_LIBS@
+libgvplugin_ming_la_LIBADD = $(top_builddir)/lib/gvc/libgvc.la @MING_LIBS@ @MATH_LIBS@
 
 if WITH_WIN32
 libgvplugin_ming_la_LDFLAGS += -no-undefined
--- graphviz-2.36.0/plugin/ming/gvrender_ming.c.orig	2014-01-11 23:59:44.000000000 +0100
+++ graphviz-2.36.0/plugin/ming/gvrender_ming.c	2014-04-06 21:17:07.658768000 +0200
@@ -91,7 +91,7 @@
 extern char* gvconfig_libdir(void);
 #define FONT "Bitstream_Vera_Serif.fdb"
 
-static void ming_textpara(GVJ_t * job, pointf p, textpara_t * para)
+static void ming_textspan(GVJ_t * job, pointf p, textspan_t * span)
 {
     SWFMovie movie = (SWFMovie)(job->context);
     SWFTextField textfield;
@@ -116,29 +116,29 @@
 
     textfield = newSWFTextField();
     SWFTextField_setFont(textfield, (SWFBlock)font);
-    SWFTextField_addChars(textfield, para->str);
-    SWFTextField_addUTF8String(textfield, para->str);
+    SWFTextField_addChars(textfield, span->str);
+    SWFTextField_addUTF8String(textfield, span->str);
     SWFTextField_setColor(textfield,
 	 pencolor.u.rgba[0],
 	 pencolor.u.rgba[1],
 	 pencolor.u.rgba[2],
 	 pencolor.u.rgba[3]);
-    SWFTextField_setHeight(textfield, para->fontsize);
+    SWFTextField_setHeight(textfield, span->font->size);
 
-    switch (para->just) {
+    switch (span->just) {
     case 'r':
 	offset.x = 0.;
 	break;
     case 'l':
-	offset.x = -para->width;
+	offset.x = -span->size.x;
 	break;
     case 'n':
     default:
-	offset.x = -para->width/2.;
+	offset.x = -span->size.x/2.;
 	break;
     }
     /* offset to baseline */
-    offset.y = -para->height + para->fontsize*.4;  /* empirically determined */
+    offset.y = -span->size.y + span->font->size*.4;  /* empirically determined */
 
     item = SWFMovie_add(movie, (SWFBlock)textfield);
     SWFDisplayItem_moveTo(item, p.x + offset.x, p.y + offset.y);
@@ -277,7 +277,7 @@
     0,				/* ming_end_anchor */
     0,				/* ming_begin_label */
     0,				/* ming_end_label */
-    ming_textpara,
+    ming_textspan,
     0,				/* ming_resolve_color */
     ming_ellipse,
     ming_polygon,
