--- graphviz-2.2/dotneato/common/cairogen-win.c.orig	2004-12-11 20:26:05.000000000 +0100
+++ graphviz-2.2/dotneato/common/cairogen-win.c	2005-02-21 11:04:40.000000000 +0100
@@ -52,8 +52,13 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
+#include <X11/Xlib.h>
+#include <X11/Xutil.h>
+#include <X11/extensions/Xrender.h>
 #include <svg-cairo.h>
+#include <cairo-xlib.h>
 
 #include "cairogen-win.h"
 
--- graphviz-2.2/dotneato/gvre_svg/cairogen-win.c.orig	2004-12-11 20:26:06.000000000 +0100
+++ graphviz-2.2/dotneato/gvre_svg/cairogen-win.c	2005-02-21 11:14:26.000000000 +0100
@@ -52,8 +52,13 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
+#include <X11/Xlib.h>
+#include <X11/Xutil.h>
+#include <X11/extensions/Xrender.h>
 #include <svg-cairo.h>
+#include <cairo-xlib.h>
 
 #include "cairogen-win.h"
 
