--- graphviz-2.30.0/cmd/smyrna/gui/gui.c.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/cmd/smyrna/gui/gui.c	2013-01-24 12:31:24.181987670 +0100
@@ -379,7 +379,7 @@ void show_gui_warning(char *str)
     Dlg = (GtkMessageDialog *) gtk_message_dialog_new(NULL,
 						      GTK_DIALOG_MODAL,
 						      GTK_MESSAGE_WARNING,
-						      GTK_BUTTONS_OK, str);
+						      GTK_BUTTONS_OK, "%s", str);
 
     respond = gtk_dialog_run((GtkDialog *) Dlg);
     gtk_object_destroy((GtkObject *) Dlg);
--- graphviz-2.30.0/cmd/smyrna/gui/menucallbacks.c.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/cmd/smyrna/gui/menucallbacks.c	2013-01-24 12:31:59.286937073 +0100
@@ -99,15 +99,13 @@ void mQuitSlot(GtkWidget * widget, gpoin
 int show_close_nosavedlg(void)
 {
     GtkWidget *dialog;
-    char buf[512];
     int rv;			/*return value */
-    sprintf(buf,
-	    "%s has been modified. Do you want to save it before closing?",
-	    view->Topview->Graphdata.GraphFileName);
     dialog =
 	gtk_message_dialog_new(NULL, GTK_DIALOG_MODAL,
 			       GTK_MESSAGE_QUESTION, GTK_BUTTONS_NONE,
-			       buf);
+	    "%s has been modified. Do you want to save it before closing?",
+	    view->Topview->Graphdata.GraphFileName);
+			       
 
     gtk_window_set_title(GTK_WINDOW(dialog), "Smyrna Warning");
     gtk_dialog_add_button((GtkDialog *) dialog, "Yes", 0);
