--- graphviz-2.12/configure.ac.orig	2006-12-27 15:09:27.103558425 +0100
+++ graphviz-2.12/configure.ac	2006-12-27 15:51:47.804344712 +0100
@@ -546,12 +546,15 @@
 	AC_MSG_WARN([swig does not support lua. The LUA packages will not be built])
 else
 AC_CHECK_PROG(LUA,lua,lua)
-LUA_INCLUDES=
+LUA_INCLUDES="-I/usr/include/lua51"
 LUA_LIBS=
+save_CPPFLAGS="$CPPFLAGS"
+CPPFLAGS="$CPPFLAGS $LUA_INCLUDES"
 AC_CHECK_HEADER(lua.h,,[
   AC_MSG_WARN([Unable to find header lua.h. The LUA packages will not be built])
   LUA=
   ])
+CPPFLAGS="$save_CPPFLAGS"
 AC_SUBST(LUA_INCLUDES)
 AC_SUBST(LUA_LIBS)
 fi
--- graphviz-2.8/tclpkg/gv/Makefile.am.orig	2006-01-10 15:39:21.000000000 +0100
+++ graphviz-2.8/tclpkg/gv/Makefile.am	2006-02-11 22:07:48.414152000 +0100
@@ -6,7 +6,7 @@
 	-I$(top_srcdir)/lib/common \
 	-I$(top_srcdir)/lib/graph \
 	-I$(top_srcdir)/lib/cdt \
-	-I$(top_srcdir)/lib/pathplan $(PERL_INCLUDES) $(OCAML_INCLUDES) $(PHP_INCLUDES) $(PYTHON_INCLUDES) $(RUBY_INCLUDES) $(TCL_INCLUDES) 
+	-I$(top_srcdir)/lib/pathplan $(PERL_INCLUDES) $(OCAML_INCLUDES) $(PHP_INCLUDES) $(PYTHON_INCLUDES) $(RUBY_INCLUDES) $(TCL_INCLUDES) $(LUA_INCLUDES)
 
 LIBS =  @LIBS@ -lc
 
