--- graphviz-2.12/configure.ac.orig	2006-12-27 15:51:47.804344000 +0100
+++ graphviz-2.12/configure.ac	2006-12-27 15:53:32.174292416 +0100
@@ -631,7 +631,7 @@
 AC_ARG_ENABLE(php,
   [AC_HELP_STRING([--disable-php], [don't support php language bindings])])
 if test "x$enable_php" != "xno"; then
-if test `$SWIG -help 2>&1 | $GREP -c '\-php *- Generate'` = 0; then
+if test `$SWIG -help 2>&1 | $GREP -c '\-php5 *- Generate'` = 0; then
 	PHP=
 	AC_MSG_WARN([swig does not support php. The PHP packages will not be built])
 else
--- graphviz-2.12/tclpkg/gv/Makefile.am.orig	2006-12-27 15:09:27.243566000 +0100
+++ graphviz-2.12/tclpkg/gv/Makefile.am	2006-12-27 15:53:44.046969001 +0100
@@ -86,7 +86,7 @@
 libgv_php_la_LIBADD = $(BASELIBS) @PHP_LIBS@
 libgv_php_la_LDFLAGS = 
 gv_php.cpp $(PHP_hdr): gv.i
-	$(SWIG) -c++ -php -o gv_php.cpp $(srcdir)/gv.i
+	$(SWIG) -c++ -php5 -o gv_php.cpp $(srcdir)/gv.i
 
 pkgpythondir = $(pkglibdir)/python
 PYTHON_hdr = gv.py
