--- graphviz-2.30.0/lib/graph/Makefile.am.orig	2013-01-20 09:19:48.263167841 +0100
+++ graphviz-2.30.0/lib/graph/Makefile.am	2013-01-20 09:21:38.306498882 +0100
@@ -11,15 +11,15 @@
 	-I$(top_srcdir)/lib/cdt
 
 man_MANS = graph.3
-if WITH_CGRAPH
-else
+#if WITH_CGRAPH
+#else
 pkginclude_HEADERS = graph.h
 noinst_HEADERS = libgraph.h triefa.h parser.h agxbuf.h
 noinst_LTLIBRARIES = libgraph_C.la
 lib_LTLIBRARIES = libgraph.la
 pkgconfig_DATA = libgraph.pc
 pdf_DATA = graph.3.pdf
-endif
+#endif
 
 libgraph_C_la_SOURCES = \
 	attribs.c agxbuf.c edge.c graph.c graphio.c lexer.c \
