--- graphviz-2.8/tclpkg/gv/gv_php.cpp.orig	2006-02-03 23:38:22.000000000 +0100
+++ graphviz-2.8/tclpkg/gv/gv_php.cpp	2006-02-11 20:13:04.627942250 +0100
@@ -704,8 +704,9 @@
   swig_module_info *ret = 0;
 
   MAKE_STD_ZVAL(pointer);
+  TSRMLS_FETCH();
 
-  if (zend_get_constant(const_name, sizeof(const_name), pointer)) {
+  if (zend_get_constant(const_name, sizeof(const_name), pointer TSRMLS_CC)) {
     if (pointer->type == IS_LONG) {
       ret = (swig_module_info *) pointer->value.lval;
     }
@@ -714,6 +715,7 @@
 }
 
 static void SWIG_Php4_SetModule(swig_module_info *pointer) {
+  TSRMLS_FETCH();
   REGISTER_MAIN_LONG_CONSTANT(const_name, (long) pointer, 0);
 }
 
@@ -759,8 +761,8 @@
 ZEND_END_MODULE_GLOBALS(gv)
 ZEND_DECLARE_MODULE_GLOBALS(gv)
 #ifdef ZTS
-#define ErrorMsg() TSRMG(gv_globals_id, zend_gv_globals *, error_msg );
-#define ErrorCode() TSRMG(gv_globals_id, zend_gv_globals *, error_code );
+#define ErrorMsg() TSRMG(gv_globals_id, zend_gv_globals *, error_msg )
+#define ErrorCode() TSRMG(gv_globals_id, zend_gv_globals *, error_code )
 #else
 #define ErrorMsg() (gv_globals.error_msg)
 #define ErrorCode() (gv_globals.error_code)
@@ -773,6 +775,7 @@
 static void gv_destroy_globals(zend_gv_globals *gv_globals) { }
 
 void SWIG_ResetError() {
+  TSRMLS_FETCH();
   ErrorMsg() = default_error_msg;
   ErrorCode() = default_error_code;
 }
