--- graphviz-2.40.1/configure.ac.orig	2017-06-01 17:38:47.623401374 +0200
+++ graphviz-2.40.1/configure.ac	2017-06-01 20:10:12.199964310 +0200
@@ -1642,7 +1642,7 @@
 	    if test "x$DARWIN" = "xyes"; then
 	      RUBY_CFLAGS=`echo $RUBY_CFLAGS | sed 's/powerpc/universal/'`
 	    fi
-            RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` `$RUBY -rrbconfig -e \"puts RbConfig::CONFIG[['LIBRUBYARG_SHARED']]\"`"
+            RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` $($RUBY -rrbconfig -e "puts RbConfig::CONFIG[['LIBRUBYARG_SHARED']]")"
             RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb vendorarchdir`"
 	    if test "x$RUBY_INSTALL_DIR" = "x"; then
 	  	  RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb sitearchdir`"
--- graphviz-2.34.0/config/config_ruby.rb.orig	2013-11-10 15:08:02.653585861 +0100
+++ graphviz-2.34.0/config/config_ruby.rb	2013-11-10 15:08:47.280250638 +0100
@@ -8,6 +8,10 @@
     puts RbConfig::expand(CONFIG["archdir"])
 when "lib"
     puts RbConfig::expand(CONFIG["libdir"])
+when "includes"
+    puts Config::expand(CONFIG["rubyhdrdir"])
+when "archincludes"
+    puts Config::expand(CONFIG["rubyhdrdir"]+"/"+CONFIG["arch"])
 when "vendorarchdir"
     puts RbConfig::expand(CONFIG["vendorarchdir"])
 when "sitearchdir"
