--- graphviz-1.10/configure.ac.orig	Fri Aug  2 22:55:20 2002
+++ graphviz-1.10/configure.ac	Sat Aug 10 18:06:08 2002
@@ -48,7 +48,7 @@
 	PATHSEPARATOR=":"
 	;;
   * ) # This sucks!  Whatever happened to standards for file system layout?
-	DEFAULT_FONTPATH="/usr/X11R6/lib/X11/fonts/TrueType:/usr/X11R6/lib/X11/fonts/truetype:/usr/X11R6/lib/X11/fonts/TTF:/usr/share/fonts/TrueType:/usr/share/fonts/truetype:/usr/openwin/lib/X11/fonts/TrueType:/usr/X11R6/lib/X11/fonts/Type1:/usr/common/share/fonts/ttf";
+	DEFAULT_FONTPATH="/usr/share/fonts/TTF:/usr/share/fonts/Type1";
 	PATHSEPARATOR=":"
 	;;
 esac
