--- graphviz-2.30.0/plugin/webp/Makefile.am.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/plugin/webp/Makefile.am	2013-01-20 18:35:17.195806496 +0100
@@ -32,7 +32,7 @@
 
 libgvplugin_webp_la_LDFLAGS = -version-info @GVPLUGIN_VERSION_INFO@
 libgvplugin_webp_la_SOURCES = $(libgvplugin_webp_C_la_SOURCES)
-libgvplugin_webp_la_LIBADD = @WEBP_LIBS@ @PANGOCAIRO_LIBS@
+libgvplugin_webp_la_LIBADD = $(top_builddir)/lib/gvc/libgvc.la @WEBP_LIBS@ @PANGOCAIRO_LIBS@
 
 if WITH_WIN32
 libgvplugin_webp_la_LDFLAGS += -no-undefined
