--- graphviz-2.30.0/plugin/visio/VisioGraphic.cpp.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/plugin/visio/VisioGraphic.cpp	2013-01-20 17:31:30.895886250 +0100
@@ -16,6 +16,7 @@
 #endif
 
 #include <algorithm>
+#include <cstring>
 
 #ifdef _MSC_VER
 #include <cfloat>
--- graphviz-2.30.0/plugin/visio/VisioText.cpp.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/plugin/visio/VisioText.cpp	2013-01-20 17:33:01.815884352 +0100
@@ -21,6 +21,7 @@
 #include "gvio.h"
 #ifdef WITH_CGRAPH
 #include <string.h>
+#include <cstdlib>
 #endif
 
 extern "C" char *xml_string(char* str);
