--- graphviz-2.30.0/plugin/visio/VisioGraphic.cpp.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/plugin/visio/VisioGraphic.cpp	2013-01-20 17:31:30.895886250 +0100
@@ -16,6 +16,7 @@
 #endif
 
 #include <algorithm>
+#include <cstring>
 
 #ifdef _MSC_VER
 #include <cfloat>
--- graphviz-2.30.0/plugin/visio/VisioText.cpp.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/plugin/visio/VisioText.cpp	2013-01-20 17:33:01.815884352 +0100
@@ -21,6 +21,7 @@
 #include "gvio.h"
 #ifdef WITH_CGRAPH
 #include <string.h>
+#include <cstdlib>
 #endif
 
 extern "C" char *xml_string(char* str);
--- graphviz-2.30.0/lib/cdt/cdt.h.orig	2013-01-14 04:11:21.000000000 +0100
+++ graphviz-2.30.0/lib/cdt/cdt.h	2013-01-20 17:35:24.789214707 +0100
@@ -17,6 +17,7 @@
 #endif
 
 #include <stddef.h>	/* size_t */
+#include <string.h>	/* memcmp, strcmp */
 
 #ifndef _BEGIN_EXTERNS_
 #define _BEGIN_EXTERNS_
@@ -238,11 +239,6 @@
 extern int		dtstat _ARG_((Dt_t*, Dtstat_t*, int));
 extern unsigned int	dtstrhash _ARG_((unsigned int, Void_t*, int));
 
-#if !_PACKAGE_ast
-extern int		memcmp _ARG_((const Void_t*, const Void_t*, size_t));
-extern int		strcmp _ARG_((const char*, const char*));
-#endif
-
 #undef extern
 _END_EXTERNS_
 
