--- graphviz-2.50.0/plugin/gtk/gvdevice_gtk.c.orig	2021-12-04 21:10:28.000000000 +0100
+++ graphviz-2.50.0/plugin/gtk/gvdevice_gtk.c	2024-07-27 14:58:45.147397887 +0200
@@ -18,13 +18,9 @@
 #include <gvc/gvplugin_device.h>
 
 #ifdef HAVE_GTK
+#include <gdk/gdk.h>
 #include <gtk/gtk.h>
 
-#include <cairo.h>
-#ifdef CAIRO_HAS_XLIB_SURFACE
-#include <X11/Xlib.h>
-#include <X11/Xutil.h>
-
 #include "interface.h"
 #include "support.h"
 
@@ -54,22 +50,25 @@
 
 static void gtk_initialize(GVJ_t *firstjob)
 {
-    Display *dpy;
-    const char *display_name = NULL;
-    int scr;
+    GdkDisplay *dpy;
+    GdkScreen *scr;
 
     gtk_set_locale ();
     gtk_init (NULL, NULL);
 
-    dpy = XOpenDisplay(display_name);
+    dpy = gdk_display_get_default();
     if (dpy == NULL) {
-        fprintf(stderr, "Failed to open XLIB display: %s\n",
-                XDisplayName(NULL));
+        fprintf(stderr, "Failed to open default display\n");
+        return;
+    }
+    scr = gdk_display_get_default_screen(dpy);
+    if (scr == NULL) {
+        fprintf(stderr, "Failed to open default screen for display: %s\n",
+			gdk_display_get_name(dpy));
         return;
     }
-    scr = DefaultScreen(dpy);
-    firstjob->device_dpi.x = DisplayWidth(dpy, scr) * 25.4 / DisplayWidthMM(dpy, scr);
-    firstjob->device_dpi.y = DisplayHeight(dpy, scr) * 25.4 / DisplayHeightMM(dpy, scr);
+    firstjob->device_dpi.x = gdk_screen_get_width(scr) * 25.4 / gdk_screen_get_width_mm(scr);
+    firstjob->device_dpi.y = gdk_screen_get_height(scr) * 25.4 / gdk_screen_get_height_mm(scr);
     firstjob->device_sets_dpi = TRUE;
 }
 
@@ -129,7 +128,6 @@
     gtk_finalize,
 };
 #endif
-#endif
 
 gvplugin_installed_t gvdevice_types_gtk[] = {
 #ifdef HAVE_GTK
