--- graphviz-2.38.0/configure.ac~	2016-03-20 11:59:28.000000000 +0200
+++ graphviz-2.38.0/configure.ac	2016-03-20 13:10:36.077403508 +0200
@@ -1110,7 +1110,7 @@
         else
           PHP_INCLUDES="-I/usr/include/php -I/usr/include/php/main -I/usr/include/php/Zend -I/usr/include/php/TSRM"
         fi
-        PHP_INSTALL_DIR="/usr/lib${LIBPOSTFIX}/php"
+        PHP_INSTALL_DIR=$(php-config --extension-dir)
         PHP_INSTALL_DATADIR="/usr/share/php"
         PHP_LIBS=
         save_CPPFLAGS=$CPPFLAGS
