--- graphviz-2.42.3/config/config_python.py~	2014-04-13 22:40:25.000000000 +0200
+++ graphviz-2.42.3/config/config_python.py	2015-12-27 14:46:09.464163577 +0100
@@ -2,11 +2,11 @@
 from distutils import sysconfig
 
 if sys.argv[1] == "archlib":
-    print(sysconfig.get_python_lib(1,1))
+    print(sysconfig.get_python_lib(plat_specific=True, standard_lib=True, prefix='/usr'))
 elif sys.argv[1] == "lib":
-    print(sysconfig.get_python_lib(0,1))
+    print(sysconfig.get_python_lib(plat_specific=False, standard_lib=True, prefix='/usr'))
 elif sys.argv[1] == "archsitelib":
-    print(sysconfig.get_python_lib(1,0))
+    print(sysconfig.get_python_lib(plat_specific=True, standard_lib=False, prefix='/usr'))
 elif sys.argv[1] == "sitelib":
-    print(sysconfig.get_python_lib(0,0))
+    print(sysconfig.get_python_lib(plat_specific=False, standard_lib=False, prefix='/usr'))
 
